lib/analytics/course_edits_csv_builder.rb

```diff
@@ -37,8 +38,14 @@ def row(revision)
     row << revision.article.full_title
     row << revision.url
     row << revision.user.username
-    row << revision.characters
+    add_character_references(revision, row)
     row << revision.new_article
     row << revision.system
   end
 end
+
+def add_character_references(revision, row)
+  row << revision.characters
+  row << revision.features['feature.wikitext.revision.ref_tags'] || 0 -
```

Comment:
Use `revision.reference_added` here instead of reimplementing the the calculation.